index.html

```diff
@@ -1034,60 +1034,107 @@
-<aside class="issue">
-  There is not currently consensus among the authors and contributors to this
-  document about the below principles.
-  Please see <a href="https://github.com/w3ctag/privacy-principles/issues/150">this issue</a>
+  There is not currently consensus among the authors and contributors to this document about the below
+  principles. Please see <a href="https://github.com/w3ctag/privacy-principles/issues/150">this
+  issue</a>
   for more details.
 </aside>
 
-<div class="practice">
+<span class="practicelab">Sites, user agents and other parties should minimize the amount of data
+about people that is shared with or transferred between parties on the Web.</span>
 
-<p><span class="practicelab" id="user-agent-data-sharing">
-User agents should only share and present information about the user that
-the user can reasonably anticipate being shared, and which directly relate to the users overt, immediate goals.
-User agents and sites should be conservative when deciding if information is related to a users goals; the
-longer the chain of reasoning used to justify data collection, the less likely for that data
-collection to be ethical and principled.
-</span></p>
+Data minimization limits the risks of data being disclosed or misused and also makes it possible to
+more meaningfully provide people with understandable decisions about data about them.
 
-</div>
-
-Examples:
+<span class="practicelab">Web APIs should be designed to minimize the amount of data that is requested
+and provide granularity and user controls over personal data that is communicated to sites.</span>
 
-* Sharing geo-location data with a map website after a user has clicked a "share my location" button has
-  a small number of "hops" between intent and data.
-* Sharing information about the user's selected DNS resolver with a site, so that the site authors can debug
-  possible issues around tail/uncommon resolvers, has a large number of "hops" between user intent (to do
-  a thing on the website) and collection motivation (users like our site, so users will want to share information
-  with us to help us improve the site).
-
-<div class="practice">
-
-<p><span class="practicelab" id="sensitive-vs-non-sensitive-data">
-User agents should generally not attempt to distinguish between sensitive and nonsensitive personal data; all
-data and personal information is likely to be sensitive to some users, often in ways unanticipated by
-browser vendors and site operators.
-</span></p>
+<aside class="note">
+This principle was further explored in an earlier TAG draft on <a
+href="https://www.w3.org/2001/tag/doc/APIMinimization-20100605.html">Data Minimization in Web
+APIs</a>.
+</aside>
 
-</div>
+In maintaining duties of <a href="#dfn-duty-of-protection">protection</a>, <a
+href="#dfn-duty-of-discretion">discretion</a> and <a href="#dfn-duty-of-loyalty">loyalty</a>, user
+agents should be conservative when sharing data not necessary to satisfy a user's immediate goals.
```

Comment:
I would like to reword this, since it seems to suggest that user agents _should_ share data that doesn't benefit users, and this (to me) still reads like it is worried mostly about site goals, and not the goals of the person using the browser.

In general, if we all agree that browsers should not share this information with sites if users do not want their browser to do so, i suggest we reword it around user intent.  WDYT of this:

```
Browsers must send some amount of information to websites in order for those sites to execute
correctly, and for users to be able to accomplish their goals in interacting with those sites.  Some
users may wish to share <em>additional data</em> with websites though, data not directly related
to the users goals in interacting with the site, but which might help the site accomplish other goals.
This non-necessary data is <dfn>ancillary data</dfn> and the purposes sites wish to use it for are
<dfn>ancillary uses</dfn>.

Browsers should only send ancillary data to sites when the browser is confident the user
wishes to send such data. Browsers should be careful, cautious and conservative when making
such decisions.

Examples of the kind of ancillary data users might wish to send to sites include information
about the users experience loading the site, the kind of browser or computing environment
the site is being executed in, or the performance of the site as the user interacts with it.

Examples of the kinds of ancillary uses sites might wish to collect such data for include:

(existing list)
```

I hope the above text accomplishes the following:
1. demonstrates there are categories of non-necessary data that sites might want to collect for benign and non-nefarious purposes
2. center the user in the decision making process, and emphasize that this data should be sent because the user wants to send it, not (only) bc the site wants it
3. address @torgo 's concern about not calling for any specific mechanism the browser should use to make sure the browser is carrying out the user's goals (e.g., avoiding overloaded terms like "asking for consent" that might suggest certain UI / UX)

Reply:
We should draw from https://www.yalelawjournal.org/feature/a-relational-theory-of-data-governance in wording this. Pete's suggested text is very individualistic, saying not to share unless it's worth it to an individual user to specifically opt into the sharing. But Professor Viljoen points out that we have collective interests in aggregating data, which should be managed socially rather than just individually. An open standards body like the W3C isn't exactly democratic, but it's closer than either just browsers or just individual users.

Reply:
As we've discussed a bit on previous calls, I don't think we want to define a category of data as ancillary data. Data can be used for multiple purposes, and it could be the same pieces of data, or a single API could be used for functional purposes on one site and for an ancillary use on another site.

(I'm trying to take other parts of this wording for an updated PR.)